File: java/jimmer-cloud/author-service/src/main/java/org/babyfish/jimmer/example/cloud/author/AuthorServiceApp.java
package org.babyfish.jimmer.example.cloud.author;

import org.babyfish.jimmer.client.EnableImplicitApi;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@EnableImplicitApi
@SpringBootApplication
public class AuthorServiceApp {

    public static void main(String[] args) {
        SpringApplication.run(AuthorServiceApp.class, args);
    }
}

File: kotlin/jimmer-cloud-kt/author-service/src/main/kotlin/org/babyfish/jimmer/example/cloud/kt/author/AuthorServiceApp.kt
package org.babyfish.jimmer.example.cloud.kt.author

import org.babyfish.jimmer.client.EnableImplicitApi
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@EnableImplicitApi
@SpringBootApplication
class AuthorServiceApp

fun main(args: Array<String>) {
    runApplication<AuthorServiceApp>(*args)
}